pkg/k8s/lookup.go

```diff
@@ -49,14 +56,36 @@ func (r *RealK8sAPI) clientSet() (*kubernetes.Clientset, error) {
 	return r.cachedClientSet, nil
 }
 
+func (r *RealK8sAPI) nodeIndexer() (cache.Indexer, error) {
+	if r.cachedNodeIndexer == nil {
+		// Create an indexer for nodes that we can use to access resource counts without
+		// going all the way to the API.
+		clientSet, err := r.clientSet()
+		if err != nil {
+			return nil, err
+		}
+
+		nodeListWatcher := cache.NewListWatchFromClient(clientSet.CoreV1().RESTClient(), "nodes", metav1.NamespaceNone, fields.Everything())
```

Comment:
Since we're only counting them, does it work if we set `fields.Nothing()`, for example?

Reply:
Not sure I understand - I believe the field selector is just to filter what events we get on the watch. In this case we don't want to filter on any particular field, so that we get all nodes.

I believe that `fields.Nothing` would match _no_ nodes, since it's `Matches()` function is hard coded to return false.